components/feature/webchannel/src/main/assets/extensions/webchannel/manifest.json

```diff
@@ -0,0 +1,17 @@
+{
+  "manifest_version": 2,
+  "name": "Mozilla Android Components - Firefox Accounts WebChannel",
+  "version": "1.0",
+  "content_scripts": [
+    {
+      "matches": ["*://*/*"],
```

Comment:
lock this to FxA domains?

Reply:
Sounds like a good idea, although that might make it harder to use custom FxA environments (stage, China, self-hosted).

Semi-related, I feel like there should also be some sort of origin check logic in here somewhere, where we verify that any messages we receive are in fact coming from the expected FxA content-server domain. Or does the webextension messaging-port magic take care of that for us?